app/assets/src/components/visualizations/table/base_table.scss

```diff
@@ -40,6 +40,14 @@
   justify-content: center;
   text-align: center;
 
+  .projectHeader {
```

Comment:
A more concise way to do this would be to edit the existing `.projectHeader` class in `projects_view.scss` and add `text-align: left`. That `.projectHeader` is already getting passed into the column's `headerClassName` in `ProjectsView.jsx` and will ensure the label is getting aligned to the left. You can see how this is done for the Samples tab! (See `.sampleHeader` in `SamplesView.jsx` and `samples_view.scss`)